deploy/backplane/srep/dynatrace/10-srep-opentelemetry.Role.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: backplane-srep-opentelemetry
+  namespace: openshift-opentelemetry-operator
+rules:
+# can view opentelemetry collectors
+- apiGroups:
+  - opentelemetry.io
+  resources:
+  - opentelemetrycollectors
```

Comment:
```suggestion
  - opentelemetrycollectors
  - instrumentations
```

Reply:
I originally had this present, but removed it because we don't expect to have any instrumentations or be interacting with them. No problem with it being included though.